client/mm/libxc/binance.go

```diff
@@ -627,11 +627,11 @@ func (bnc *binance) Connect(ctx context.Context) (*sync.WaitGroup, error) {
 	wg := new(sync.WaitGroup)
 
 	if err := bnc.getCoinInfo(ctx); err != nil {
-		return nil, fmt.Errorf("error getting coin info: %v", err)
+		return nil, fmt.Errorf("error getting coin info: %w", err)
```

Comment:
These aren't wrong. You can wrap in an error. You can't wrap in a `log.Errorf`. Where' is the `%w` in a `log.Errorf` that you saw?

Reply:
Right, should be fixed now